source/docs/appendices/release-notes.md

```diff
@@ -115,31 +115,31 @@ Requirements: same as [2.4.0](#2-4-0)
 
 Requirements: same as [2.4.0](#2-4-0)
 
-## 2.4.6
+## 2.4.6 (2021-03-17)
 
 > This release contains a bugfix of guest orders not being tracked.
 
 - [Changelog](https://gitlab.com/front-commerce/front-commerce/-/releases/2.4.6)
 
 Requirements: same as [2.4.0](#2-4-0)
 
-## 2.4.5
+## 2.4.5 (2021-03-03)
 
 > This release contains a bugfix that improves resilience in case of ElasticSearch downtimes.
 
 - [Changelog](https://gitlab.com/front-commerce/front-commerce/-/releases/2.4.5)
 
 Requirements: same as [2.4.0](#2-4-0)
 
-## 2.4.4
+## 2.4.4 (2021-02-24)
 
 > This release contains minor bugfixes related to displaying invoices and dropdown.
 
 - [Changelog](https://gitlab.com/front-commerce/front-commerce/-/releases/2.4.4)
 
 Requirements: same as [2.4.0](#2-4-0)
```

Comment:
Those anchors are broken… could you please check if there are other occurrences of `](#2` in the documentation? (not a huge deal to have them broken in this page, but may make sense at other locations)

Reply:
fixed and no other links to the changed anchors